src/peer_digest.cc

```diff
@@ -667,85 +647,63 @@ peerDigestFetchStop(DigestFetchState * fetch, char *buf, const char *reason)
 {
     assert(reason);
     debugs(72, 2, "peerDigestFetchStop: peer " << fetch->pd->host << ", reason: " << reason);
-    peerDigestReqFinish(fetch, buf, 1, 1, 1, reason, 0);
+    peerDigestReqFinish(fetch, buf, reason, 0);
 }
 
-/* call this when all callback data is valid but something bad happened */
+/// diff reducer: handle a peer digest fetch failure
```

Comment:
IMO these should be labeled as TODO, or since it is obvious that this is just a wrapper:

```suggestion
/// handle a peer digest fetch failure
```